### DISCUSSION TITLE
(Feature request) Windows 10's Action Center

### DISCUSSION THREAD

**Discussion by cairopega** [valinet/ExplorerPatcher#945] [Ideas]
Bring the ability to change between Windows 11 and Windows 10's Action Center

**Comment by GitStudying** [comment]
I don't necessarily need this myself, but a lot of friends do miss the Windows 10 action/notification center. If this is still possible and the components are still in Windows 11, would you consider adding this feature?

**Comment by XuanChen1234** [comment]
I Love Windows 10 action/notification center.
![Screenshot 2023-08-02 125855](https://github.com/valinet/ExplorerPatcher/assets/140480110/f5179409-c088-49ce-b12d-a980ac2631d6)

**Reply by MinerAC4** [reply]
I agree. I [bbmaster123](https://github.com/bbmaster123) had actually had an unstable version of this working on windows 11 that worked with their old start menu mod. It often broke, but it showed that this could possibly be implemented eventually!
![Action center](https://github.com/valinet/ExplorerPatcher/assets/99057951/a2e81c80-4c4b-4dcc-a77e-fffead16b2b7)

**Reply by JVAviationPhotography** [reply]
Where did you get this mod? Does it still work? I'd love the old action centre in Windows was clean everything was in one spot the new one is just garbage. I also customised my Windows 11 install to look like windows 10 using the ExplorerPatcher Mod and using windows 10 sounds i have from a previous build.

**Reply by MinerAC4** [reply]
Unfortunately this mod no longer works, because it worked in conjunction the old start menu mod that [bbmaster123](https://github.com/bbmaster123) had created, and this mod no longer works in newer builds of windows 11. It was also highly unstable and crashed a lot of the time. I would only have recommended it for experimental purposes to begin with.

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio
